src/raptor_libxml.c

```diff
@@ -822,9 +823,7 @@ raptor_libxml_xmlStructuredError_handler_common(raptor_world *world,
   }
 
   nmsg = (char*)raptor_stringbuffer_as_string(sb);
-  if(err->level == XML_ERR_FATAL)
-    level = RAPTOR_LOG_LEVEL_FATAL;
-  else if(err->level == XML_ERR_ERROR)
+  if(err->level == XML_ERR_FATAL || err->level == XML_ERR_ERROR)
```

Comment:
This change squashes xml fatal errors down to raptor error level - was there a reason  for this?